src/components/manage_users_table/ManageUsersTable.js

```diff
@@ -0,0 +1,190 @@
+import { useState, useEffect, Fragment, useCallback } from 'react';
+import { h } from 'react-hyperscript-helpers';
+import {isNil} from 'lodash/fp';
+import { Styles } from '../../libs/theme';
+import PaginationBar from '../PaginationBar';
+import { recalculateVisibleTable, goToPage as updatePage, getSearchFilterFunctions, searchOnFilteredList } from '../../libs/utils';
+import SimpleTable from '../SimpleTable';
+import cellData from './ManageUsersTableCellData';
+
+export const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.6rem',
+    fontWeight: 400,
+    display: 'flex',
+    padding: '1rem 2%',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    whiteSpace: 'pre-wrap',
+    backgroundColor: 'white',
+    border: '1px solid #DEDEDE',
+    borderRadius: '4px',
+    margin: '0.5% 0'
+  },
+  columnStyle: Object.assign({}, Styles.TABLE.HEADER_ROW, {
+    justifyContent: 'space-between',
+    color: '#7B7B7B',
+    fontFamily: 'Montserrat',
+    fontSize: '1.2rem',
+    fontWeight: 'bold',
+    letterSpacing: '0.2px',
+    textTransform: 'uppercase',
+    backgroundColor: 'B8CDD3',
+    border: 'none'
+  }),
+  cellWidth: {
```

Comment:
**Optional** 

So the skeleton loader looks a bit awkward because the column widths are 33.33%. Since the loader just uses the same values for the graphic the bars overlap with each other. If you give the the cellWidth some space in between (and update the columnStyle appropriately, the loaders will look a lot cleaner.

One example: Adjusting the cellWidths so that the sum is equal to `94%` gives you `6%` space for margins, or `1%` on each side of a column. From there you can just update the `cellStyles` on `columnHeaderConfig` so that its something like `cellStyle: {width: <insert width here>, margin: '0% 1%'}`

Reply:
good catch!